src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/VerifiedUserClaimTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SAF.DAS.ApprenticeCommitments.Web.UnitTests.AutoFixtureCustomisations;
+using SFA.DAS.ApprenticeCommitments.Web.Services;
+using SFA.DAS.ApprenticeCommitments.Web.Services.OuterApi;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.UnitTests
+{
+    public class VerifiedUserClaimTests
+    {
+        [Test, MoqAutoData]
+        public async Task Users_without_an_apprentice_id_do_not_have_VerifiedUser_claim(AuthenticationEvents sut, ClaimsPrincipal identity)
+        {
+            await sut.AddUserVerifiedClaim(identity);
+
+            identity.Claims.Should().NotContain(c => c.Type.Equals("VerifiedUser", StringComparison.OrdinalIgnoreCase));
+        }
+
+        [Test, MoqAutoData]
+        public async Task Users_with_an_invalid_apprentice_id_do_not_have_VerifiedUser_claim(AuthenticationEvents sut, ClaimsPrincipal identity, string notAGuid)
+        {
+            identity.AddIdentity(new ClaimsIdentity(new[] { new Claim("apprentice_id", notAGuid) }));
+
+            await sut.AddUserVerifiedClaim(identity);
+
+            identity.Claims.Should().NotContain(c => c.Type.Equals("VerifiedUser", StringComparison.OrdinalIgnoreCase));
+        }
+
```

Comment:
We are giving ourselves a bit of a maintenance issue with the "VerifiedUser" and "apprentice_id" claim names.

Reply:
Do you mean having the string literals?  I agree that the constants make sense in the main codebase.  I'm not convinced they make as much sense in the tests.

Having the actual value shown in the tests makes it more explicit what's actually being used I think.  And if we were to ever change the name of the claim we'd probably rename the constant as well. which would change all these lines anyway.

I could probably be pursuaded if you feel strongly about it.

Reply:
no i do agree, the literal string in tests make sense